adyendev-postman-release-notes.md

```diff
@@ -0,0 +1,40 @@
+# AdyenDev Release Notes
+
+The AdyenDev Postman space helps developers to explore, integrate and test the Adyen APIs.  
+
+This page contains the **Release Notes** of the following resources:
+* [Adyen APIs](#adyen-apis)
+* [Adyen in-person payments](#adyen-in-person-payments)
+* [Adyen Postman Flows](#adyen-postman-flows)
+
+## Adyen APIs
+
+> All Adyen APIs with documentation, variables and examples. See on [Postman](https://www.postman.com/adyendev/workspace/adyen-apis/overview).
+
+### Nov 2023
+- The path variables are defined using the Postman conventions, for example `/paymentLinks/:linkId`. This allows to set the value of the path parameter without editing the URL.  
+- The endpoints within a folder are ordered alphabetically.  
+
+### Sept 2023
+- The path variables (i.e. `/action/{id}`) are no longer defined as environment variables. Edit them directly in the endpoint URL for more flexibility, enabling the testing of various scenarios and entities.
+
+### Feb 2023
+- First release: creation of the workspace and all Postman collections.
+
+---
+
+## Adyen in-person payments
+
+> Build a feature-rich point-of-sale integration with the Adyen in-person payments solution. See on [Postman].(https://www.postman.com/adyendev/workspace/adyen-in-person-payments/overview).
```

Comment:
The ipp-team rebranded from Point-of-sale to In-person Payments, let's use In-person payments everywhere for consistency w/ docs